From 7bbf88bfd6b5200926294788386398193afe623f Mon Sep 17 00:00:00 2001
From: Cole Robinson <crobinso@redhat.com>
Date: Sat, 4 Jun 2022 20:28:58 -0400
Subject: [PATCH] Disable flakey dbus-display-test

Signed-off-by: Cole Robinson <crobinso@redhat.com>
---
 tests/qtest/meson.build | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/tests/qtest/meson.build b/tests/qtest/meson.build
index d25f82bb5a..d085604727 100644
--- a/tests/qtest/meson.build
+++ b/tests/qtest/meson.build
@@ -94,10 +94,6 @@ qtests_i386 = \
    'test-filter-redirector'
   ]
 
-if dbus_display
-  qtests_i386 += ['dbus-display-test']
-endif
-
 dbus_daemon = find_program('dbus-daemon', required: false)
 if dbus_daemon.found() and config_host.has_key('GDBUS_CODEGEN')
   # Temporarily disabled due to Patchew failures:
@@ -298,10 +294,6 @@ qtests = {
   'vmgenid-test': files('boot-sector.c', 'acpi-utils.c'),
 }
 
-if dbus_display
-qtests += {'dbus-display-test': [dbus_display1, gio]}
-endif
-
 qtest_executables = {}
 foreach dir : target_dirs
   if not dir.endswith('-softmmu')
-- 
2.37.0.rc2

